teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here you need to send the person constructor parameters in the teacher constructor parameters.

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
end
```

Reply:
Okay, thank you. I will work on it.